components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -108,22 +116,33 @@ func main() {
 	fullRuntimeComponentList, err := runtimeProvider.AllComponents()
 	fatalOnError(err)
 
+	gardenerClusterConfig, err := newGardenerClusterConfig(cfg)
+	fatalOnError(err)
+	//
+	gardenerSecrets, err := newGardenerSecretsInterface(gardenerClusterConfig, cfg)
```

Comment:
please use existing kuberrnetes client, do not create another one
above in this file k8s cleint is creted:
```
k8sCfg, err := config.GetConfig()
fatalOnError(err)
cli, err := client.New(k8sCfg, client.Options{})
fatalOnError(err)
```